src/nodes.jl

```diff
@@ -335,13 +332,7 @@ function run!(op::Op)
         end
     end
 
-    kwargs = map(op.kwargs) do kwarg
-        if isa(kwarg.second, DispatchNode)
-            return (kwarg.first => deps[kwarg.second])
-        else
-            return kwarg
-        end
-    end
+    kwargs = [kwarg=> isa(kwarg.second, DispatchNode) ? kwarg.first => deps[kwarg.second] : kwarg for kwarg in op.kwargs]
```

Comment:
Why change this? I find the `map` clearer, personally.

Reply:
I think the `map` output on 0.7 was causing an error later on. This is just a stopgap measure until I figure out what was going wrong with it. I also find the `map` clearer.

Reply:
Might have to do the map on `pairs(op.kwargs)` (if that works)

Reply:
Keyword arguments always iterate name-value pairs, so `pairs` shouldn't make a difference. Actually, IIRC, in 0.7 and 1.0, keyword arguments are made available inside the function as the same kind of iterator produced by the `pairs` function.